themes/tiddlywiki/vanilla/base.tid

```diff
@@ -555,6 +555,10 @@ html body.tc-body .tc-btn-rounded:hover svg {
 	fill: <<colour muted-foreground>>;
 }
 
+svg + .tc-btn-text, .tc-image-button + .tc-btn-text{
+	margin-left: 0.3em;
```

Comment:
0.3em seems / feels a little bit close. Did you try 0.4 or 0.5. ... How does that feel?

Reply:
does the CSS need to be so complicated?

Isn't it enough to use the rule below. ... IMO it should be

Reply:
If we have it as you suggest then it will run always, with or without image. The sibling selector avoids it. https://developer.mozilla.org/en-US/docs/Web/CSS/Adjacent_sibling_combinator

And the rule `.tc-btn-text` looks unused.

If finally the value is `0.5em`, the selector maybe could be add in this rule 
https://github.com/BurningTreeC/TiddlyWiki5/blob/62eee8118368b7f728b00537d056cd885194ace5/themes/tiddlywiki/vanilla/base.tid#L2774

Reply:
I think you are right that just using `.tc-btn-text` would be good. Currently, `.tc-btn-text` is always preceded by an icon, so I'll assume that it is a class that is used for button texts that comes after icons. It can simply be changed if someone uses the class for other purposes.

Also, setting margin-left to an `em` value did not look good on large font-sizes. (Which can be seen for ViewToolbar when you set `$:/config/Toolbar/Text` to true.) So switched to pixel value `7px` which is approximately  `0.4em` for the dropdowns.

Reply:
>I think you are right that just using .tc-btn-text would be good. Currently, .tc-btn-text is always preceded by an icon, so I'll assume that it is a class that is used for button texts that comes after icons. It can simply be changed if someone uses the class for other purposes.

You are right, it is exclusively used that way atm. So imo we can use it to add the margin-left, if possible. I would like to avoid CSS selectors that are too specific. If we can use `.tc-btn-text` we should use it.

Reply:
We need to make sure, that the following setting looks good: https://tiddlywiki.com/#%24%3A%2Fcore%2Fui%2FControlPanel%2FSettings%2FToolbarButtons

Check the "Include Text" box and you will see, what I mean. 

I personally think, that this always looks ugly. ... May be in a second PR we should try to make the text a bit smaller and see, how it looks like

Reply:
I think the toolbar looking ugly when 'Include Text' is turned on, is a different issue and out of scope for this PR. It should be done in another PR, probably by someone who has more design sense than me :)